### PR TITLE
Utilize github.com/jaypipes/pcidb

### PR DESCRIPTION
**Pull request by jaypipes**
The PCI-IDS database querying and parsing functionality has been pulled
out of `ghw` into the github.com/jaypipes/pcidb library. This patch
simply removes the corresponding functionality that deal with the
PCI-IDS database parsing and adds the dependency on pcidb to the ghw
library.

Issue #43

### CODE REVIEW

**Comment by jaypipes** [comment]
@retr0h mind taking a gander at this one and seeing if anything pops out as bad?

**Comment by retr0h** [comment]
Sure

File: pci.go [hunk 1]
```diff
@@ -18,46 +20,14 @@ var (
...
 	Address              string // The PCI address of the device
-	Vendor               *PCIVendor
-	Product              *PCIProduct
-	Subsystem            *PCIProduct // optional subvendor/sub-device information
-	Class                *PCIClass
-	Subclass             *PCISubclass             // optional sub-class for the device
-	ProgrammingInterface *PCIProgrammingInterface // optional programming interface
+	Vendor               *pcidb.PCIVendor
+	Product              *pcidb.PCIProduct
+	Subsystem            *pcidb.PCIProduct // optional subvendor/sub-device information
```
Comment (retr0h) [inline pci.go]: go fmt somehow not fmt the comments on the same spacing as below?

Comment (jaypipes) [reply pci.go]: yeah, weird, huh? I have go fmt run when I save in vim and for some reason it's lining up the types but not the comments. oh well...

**Comment by jaypipes** [comment]
thanks for the review, @retr0h! :)

**Pull request merged**